src/centrality/pagerank.jl

```diff
@@ -7,11 +7,12 @@ iterations (`n`), and convergence threshold (`ϵ`). If convergence is not
 reached within `n` iterations, an error will be returned.
 """
 function pagerank(g::DiGraph, α=0.85, n=100, ϵ = 1.0e-6)
-    M = adjacency_matrix(g,:out,Float64)
-    S = vec(sum(M,1))
+    A = adjacency_matrix(g,:out,Float64)
+    S = vec(sum(A,1))
     S = 1./S
     S[find(S .== Inf)]=0.0
-    M = scale(S, M')
+    M = A' # need a separate line due to a bug in julia
```

Comment:
Which bug is this? Will it be fixed by the v0.5.0 release?

The following loop on line 24 appears to transpose the matrix on every loop iteration. Can we just store the transpose here and get a huge speedup?